src/planwise/endpoint/projects2.clj

```diff
@@ -18,6 +18,14 @@
   [project]
   (dissoc project :deleted-at))
 
+(defn- filter-projects-owned-by
+  [projects owner-id]
+  (filter (fn [project] (= (:owner-id project) owner-id)) projects))
+
+(defn- filter-owned-by
+  [project owner-id]
+  (first (filter-projects-owned-by [project] owner-id)))
```

Comment:
This is backwards :) You should use build a predicate for a single project and use that in the case of collections.